src/core_plugins/metrics/public/components/panel_config/markdown.js

```diff
@@ -2,7 +2,6 @@ import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import SeriesEditor from '../series_editor';
 import { IndexPattern } from '../index_pattern';
-import AceEditor from 'react-ace';
 import 'brace/mode/less';
```

Comment:
Is this still needed? It feels like this should _only_ happen within the `KuiCodeEditor` abstraction

Reply:
Good question. If we pull this inside `KuiCodeEditor` which modes do we import? All; just the ones we need at the moment, and people using this with new languages, will need to change `KuiCodeEditor` too?

Again: I would like to provide a proper code editor at some point, where this should be cleaned up, but I guess not in this first step. (We apparently also include some modes in webpackShims...)